.github/pull_request_template.md

```diff
@@ -33,6 +33,8 @@ Please include any screenshots that communicate the visual story of the change t
 
 ### Notes
 
+Recommended reading: [Code Review guide](https://github.com/customink/guides/blob/master/operations/code-review/README.md)
+
```

Comment:
Should this be wrapped in a comment? If not, it'll be rendered in the description if the person opening the PR doesn't delete it.

Reply:
I thought the point was to have it display so everyone can easily reference it while viewing a PR

Reply:
OK, fair enough. I guess my concern is it might distract from "actual" notes someone leaves, specific to the PR.